### PR TITLE
made the top bar's orientation proper after logging in

### PR DESCRIPTION
**Pull request by Sloth-Panda**
corresponds to issue-> #57

### CODE REVIEW

File: main_app/mail.py [hunk 1]
```diff
@@ -14,16 +14,16 @@ def send_email(name, dest, link):
     server.starttls()
 
     # Enter your Email and Password
-    server.login("Email ID", "Password")
+    server.login("achyut662cs@gmail.com", "662cs@12345")
```
Comment (sanjay270899) [inline main_app/mail.py:17]: Remove your Email ID and Password, and change it "Email ID" and "Password" respectively.

File: main_app/mail.py [hunk 2]
```diff
@@ -14,16 +14,16 @@ def send_email(name, dest, link):
...
     email_html = open('main_app/templates/main_app/email.html')
     email_body = email_html.read().format(name=name, link=link)
     msg = MIMEMultipart()
     msg['Subject'] = 'EMERGENCY'
     msg.attach(MIMEText(email_body, 'html'))
     
     # Again enter your Email ID
-    msg['From'] = formataddr(("TEAM RESCUE", "Email ID"))
+    msg['From'] = formataddr(("TEAM RESCUE", "achyut662cs@gmail.com"))
```
Comment (sanjay270899) [inline main_app/mail.py:25]: Here also. @Sloth-Panda

File: main_app/mail.py [hunk 3]
```diff
@@ -14,16 +14,16 @@ def send_email(name, dest, link):
...
 
     # One last time add your email
-    server.sendmail("Email ID", dest, msg.as_string())
+    server.sendmail("achyut662cs@gmail.com", dest, msg.as_string())
     server.quit()
```
Comment (sanjay270899) [inline main_app/mail.py:29]: Here also.

**Review by sanjay270899 — CHANGES_REQUESTED** [review]

**Comment by Sloth-Panda** [comment]
oh ,  forgot to remove that

**Comment by sanjay270899** [comment]
@Sloth-Panda I will suggest changing the password of your Gmail account right now.

**Comment by Sloth-Panda** [comment]
@sanjay270899 have corrected that

**Comment by sanjay270899** [comment]
@Sloth-Panda Now it is fine. Also, I will suggest you to change the password of your google account.

**Comment by Sloth-Panda** [comment]
@sanjay270899 changing still delete the photos

**Comment by swapnilsparsh** [comment]
Issue has been resolved #67

**Pull request closed without merging**